### PR TITLE
Telemetry misconfiguration test

### PR DESCRIPTION
**Pull request by Yas-Franco**
For issue #24073

### CODE REVIEW

**Comment by Yas-Franco** [comment]
#libby
#build

**Comment by yasmin-aumeeruddy** [comment]
#build
#libby

**Review by benjamin-confino — DISMISSED** [review]

File: dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/TelemetryMisconfigTest.java [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+                        .run(String.class);
+    }
+
+    @Test
+    @ExpectedFFDC({ "org.jboss.resteasy.spi.UnhandledException" })
+    public void testInvalidJaegerTimeoutEndpoint() throws Exception {
+        String createSpan = new HttpRequest(server, "/" + INVALID_JAEGER_TIMEOUT_APP_NAME + "/misconfig/jaxrsclient")
+                        .expectCode(500)
+                        .run(String.class);
+    }
```
Comment (Azquelt) [inline dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/TelemetryMisconfigTest.java]: Should each of these tests not be looking for an error message in the logs?

**Review by Azquelt — CHANGES_REQUESTED** [review]

File: dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/TelemetryMisconfigTest.java (hunk 1 above)
Comment (Azquelt) [reply dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/TelemetryMisconfigTest.java]: I can't see where anything is testing for an expected error message.

**Comment by Azquelt** [comment]
That's much better thank you. However, I notice that your invalid endpoints are strings like "INVALID_ENDPOINT" - could you add a test where the endpoint looks valid but there's no server listening there?

Given the error messages we're expecting, I think that would be handled differently.

File: dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/apps/jaxrsmisconfig/JaxRsMisConfigEndpoints.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
+/*******************************************************************************
+ * Copyright (c) 2022 IBM Corporation and others.
```
Comment (Azquelt) [inline dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/apps/jaxrsmisconfig/JaxRsMisConfigEndpoints.java]: Copyright date

**Comment by Yas-Franco** [comment]
#libby
#build

**Review by Azquelt — DISMISSED** [review]

**Comment by Yas-Franco** [comment]
The build failed as the [NotKnownEndpointTest](https://github.com/yasmin-aumeeruddy/open-liberty/blob/telemetry-misconfiguration-test/dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/TelemetryMisconfigTest.java#L184) checked for an error message that changed depending on the OS used. The test has been updated

**Comment by Yas-Franco** [comment]
#libby
#build

**Review by benjamin-confino — APPROVED** [review]

**Pull request merged**